File: trunk/WebProject/web/script/published.js
/**
 * Valida o formulário dos artigos publicados.
 */
function validateForm()
{
    var form = document.forms["form"];
    
    return validate(form["completeName"].value, /[a-z0-9 ]{4,40}$/i, 
	"DOI, ISSN ou parte do nome do periódico inválido.");
}

File: trunk/WebProject/src/java/webproject/bean/Published.java
package webproject.bean;

public class Published extends LoginDependant
{

    private String completeName;

    public Published()
    {
	attributes.put(new Bean.Attribute("completeName",  1, "DOI, ISSN ou parte do nome do periódico"), "getProfInstitution");
    }

    public String getCompleteName()
    {
	return completeName;
    }

    public void setCompleteName(String completeName)
    {
	this.completeName = completeName;
    }
}
